Problem1.java

```diff
@@ -0,0 +1,85 @@
+// Time Complexity : O(logn)
+// Space Complexity : O(1)
+// Did this code successfully run on Leetcode : Time Limit Exceeded
+// Any problem you faced while coding this : How to get the pivot in LogN time
+
+
+// Your code here along with comments explaining your approach
+
```

Comment:
You are trying to do too many operations to find the target value.
As taught in class, watch for on which side of the mid you have your target value and process with binary search further.